camel/toolkits/browser_toolkit.py

```diff
@@ -126,6 +127,16 @@ class InteractiveRegion(TypedDict):
     rects: List[DOMRectangle]
 
 
+class AcceptedBrowserChannels(Enum):
```

Comment:
Is this specific to chromium? Maybe a shorter naming such as `ChromiumChannels`

Reply:
For now, we only support chromium and its distro. However, `playwright` supported `firefox` and `webkit`. In the feature, if we or someone else need to extend it, they can work from here. Therefore, I would suggest the name `AcceptedBrowserChannels`. What do you think @zechengz ?

Reply:
If you don't have any more feedback about this, please don't hesitate to let me know. I would like to close this conversation.

Reply:
IMO firefox etc is something like `BrowserType` https://playwright.dev/docs/api/class-browsertype. The `AcceptedBrowserChannels` seems only apply to the `chromium`?

Reply:
Good point, I've changed it to `ChromiumChannels`.